testutil/client.go

```diff
@@ -124,7 +124,7 @@ func init() {
 	SockAddrZero4 = ContainerAddr("zero2", 5080)
 	SockAddrZero4Http = ContainerAddr("zero2", 6080)
 
-	fmt.Printf("testutil: %q %s %s\n", DockerPrefix, SockAddr, SockAddrZero)
+	// fmt.Printf("testutil: %q %s %s\n", DockerPrefix, SockAddr, SockAddrZero)
```

Comment:
why are we removing this?

Reply:
We were printing these logs for testing purposes, and they were printed in the init function. I'm not sure how we can avoid them when the user tries the checkupgrade command. For now, I've temporarily commented them out. If I find a solution, I'll apply it. If you have any ideas on how we can avoid these logs, please feel free to share.

Reply:
can remove this now!